src/eckit/geo/Figure.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * (C) Copyright 1996- ECMWF.
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * In applying this licence, ECMWF does not waive the privileges and immunities
+ * granted to it by virtue of its status as an intergovernmental organisation nor
+ * does it submit to any jurisdiction.
+ */
+
+
+#pragma once
+
+#include <string>
+
+#include "eckit/memory/Builder.h"
+#include "eckit/memory/Factory.h"
+
+
+namespace eckit::geo {
+class Spec;
+}
+
+
+namespace eckit::geo {
+
+
+class Figure {
```

Comment:
It is not obvious to me what a Figure means in `geo` context. Perhaps a few lines of comments with documentation could be helpful

Reply:
Done